### ISSUE TITLE
Fatal errors during login 

### ISSUE THREAD

**Issue by kp52** [ClipperCMS/ClipperCMS#104]
Installed Clipper 1.1 and DBAPI_use from latest branches locally on 9th December, and found a couple of old problems have reappeared - premature call to $modx object in settings.inc.php, and syntax error due to extra bracket in log.class.inc.php. 

Previously I moved settings.inc,.php down in login.processor.php (https://github.com/ClipperCMS/ClipperCMS/pull/83#commitcomment-2207128) to avoid this, yesterday I moved the call to make a new document parser object up above it; alternative is to revert to mysql instead of DBAPI in settings.inc.php. I haven't fully checked out consequences of the moves.

**Comment by kp52** [comment]
I think I've worked out what happened here - my local GitHub for Windows branch differed from the version here. When I did "git fetch upstream" then "git merge upstream/1.1.develop", my local changes were preserved. I needed to do "git reset --hard upstream/1.1.develop" to get the ClipperCMS branch as currently on github.com into my local working copy. Then I've done "git push -f origin  1.1.develop" to get my GitHub repo in sync.
